index.html

```diff
@@ -14,18 +14,31 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
   </head>
   <body>
-    <header>
+    <header class="header-container active">
+      <h2 class="my-logo">My logo</h2>
+      <button  type="button" id="menu-bar"><i class="fa fa-bars"></i></button>
```

Comment:
- [ ] Great job for implementing the suggestions by the previous reviewer. However, one of the suggestions by the previous reviewer has not been implemented. When the hamburger icon is clicked, the menu items appear with the blur background which is the expected behaviour. But the menu items and the background begin to disappear when the page is scrolled down. The menu items and the background are supposed to remain on the page while the content of the page under can be scrolling except a menu item or close button is clicked. Please, take a look and fix the issue. 

[blur-move.webm](https://user-images.githubusercontent.com/26597432/215697165-27fd87aa-795b-442c-88b1-f5e087968ed8.webm)

- [OPTIONAL] I would also recommend you give margin left and right to your page in order to appear exactly like figma desing. Please, take a look.

### Your Design
![your-headline](https://user-images.githubusercontent.com/26597432/215697783-b356f9af-da0f-435a-bf1d-01e2cc734f46.png)

### Figma Design 
![temp-headline](https://user-images.githubusercontent.com/26597432/215697823-16ff1708-532d-4b2e-90ec-80143be5800c.png)

Reply:
Respected sir,
There is a small question .Should my menu bar cover the entire portfolio?

Reply:
Your pop page should cover the visible height of your page until an action is performed to remove the pop page. As you're scrolling, the pop page would remain visible to the height of the page until it is removed.

Reply:
Thank you sir

Reply:
You're most welcome :+1: